Legion/ArtifactScenarios/Kruul.lua

```diff
@@ -85,7 +85,8 @@ function mod:OnBossEnable()
 	self:Log("SPELL_CAST_START", "DrainLife", 234423)
 	self:Log("SPELL_CAST_START", "HolyWard", 233473)
 	self:Log("SPELL_AURA_APPLIED_DOSE", "AuraOfDecay", 234422)
-	self:Log("SPELL_CAST_START", "Smash", 234631)
+	self:Log("SPELL_CAST_START", "Smash", 241717)
+	self:Log("SPELL_CAST_START", "Smash", 236537)
```

Comment:
the way to do this would be placing both spellIds in the same line for the 1 Smash event.